### PR TITLE
Update rosettanet-connector-reference.adoc

### PR DESCRIPTION
**Pull request by gastonbritomulesoft**

### CODE REVIEW

File: modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc [hunk 1]
```diff
@@ -1,160 +1,163 @@
...
-| Object store reference a| ObjectStore |  Object store configuration reference. If not set the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries; if set, the referenced bean must be an object store configuration to be used instead. |  |
-| Mule DUNS identifier a| String |  DUNS business identifier for this organization. |  |x
-| Mule location identifier a| String |  Location ID for this organization. |  |x
-| Partner DUNS identifier a| String |  Expected DUNS business identifier for partner organization. |  |x
-| Partner location identifier a| String |  Expected location ID for partner organization. |  |x
+** Production |  Choose whether operating in Test or Production mode, or if both allowed.|  Production|
+| Object store reference a| Object Store |  Object store configuration reference. If not set the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retrys. If set, the referenced bean must be an object store configuration to use instead. |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc]:
```suggestion
| Object store reference a| Object Store |  Object store configuration reference. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced bean must be an object store configuration. |  |
```

File: modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc [hunk 2]
```diff
@@ -1,160 +1,163 @@
...
 
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Primary Node Only a| Boolean |  Whether this source should only execute on the primary node when running in a Cluster. |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster. |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc]:
```suggestion
| Primary Node Only a| Boolean |  Whether this source should be executed only on the primary node when running in a cluster. |  |
```

File: modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc [hunk 3]
```diff
@@ -1,160 +1,163 @@
...
```
Comment (valkyrie69) [inline modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc]:
```suggestion
| Primary Node Only a| Boolean |  Whether this source should be executed only on the primary node when running in a cluster. |  |
```

**Review by mcherem — APPROVED** [review]

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc]: Fixed.

**Pull request merged**